server/item/boots.go

```diff
@@ -60,5 +60,23 @@ func (b Boots) Boots() bool {
 
 // EncodeItem ...
 func (b Boots) EncodeItem() (name string, meta int16) {
-	return "minecraft:" + b.Tier.Name + "_boots", 0
+	return "minecraft:" + b.Tier.Name() + "_boots", 0
+}
+
+// DecodeNBT ...
+func (b Boots) DecodeNBT(data map[string]any) any {
+	if t, ok := b.Tier.(ArmourTierLeather); ok {
+		if v, ok := data["customColor"].(int32); ok {
+			t.Colour = rgbaFromInt32(v)
+		}
+	}
+	return b
+}
+
+// EncodeNBT ...
+func (b Boots) EncodeNBT() map[string]any {
+	if t, ok := b.Tier.(ArmourTierLeather); ok {
+		return map[string]any{"customColor": int32FromRGBA(t.Colour)}
```

Comment:
I think this `customColor` is not supposed to be written if `(t.Colour == color.RGBA{})`.